docs/howto/overriding-shell.txt

```diff
@@ -0,0 +1,47 @@
+=========================
+How to override the shell
+=========================
+
+Django provides functionality to auto-import project models in the shell. In your project, you might want to either avoid importing models or include additional imports in the shell. You can do this by subclassing the :djadmin:`shell` command and overriding a method.
```

Comment:
Wrap lines at 80 characters: https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#guidelines-for-restructuredtext-files

Most text editors allow configuring a line to appear in that column, so you can visually wrap it. You can also find automatic wrapping commands or extensions for most editors.